packages/govuk-frontend/src/govuk/components/footer/footer.yaml

```
@@ -2,24 +2,24 @@ params:
   - name: meta
     type: object
     required: false
-    description: Object containing options for the meta navigation.
+    description: The section to add content before the standard footer component.
```

I think I understand the intent here – that the 'standard footer' is the copyright and license information? But adding the word _component_ implies it's the entire thing (i.e. the meta section is before the grey block that makes up the footer).

I think people's mental model of what the 'standard footer' is will differ, and some will include any navigation etc that appears within it.

Can we better describe this? Maybe something similar to what we have for some of the other sections, like 'after any navigation, before the copyright and license information'?

---

Sounds like a good suggestion for `meta`

I'll do the same with the "standard footer" mention in `navigation` too

```suggestion
    description: The meta section of the footer after any navigation, before the copyright and license information.
```